db/migrate/20180105192756_rename_rules_to_subject_rules.rb

```diff
@@ -0,0 +1,12 @@
+class RenameRulesToSubjectRules < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :rule_effects, :rule_id
+    remove_foreign_key "rule_effects", "rules"
+
+    rename_table :rules, :subject_rules
```

Comment:
Renaming the actual table would break the old Sidekiq workers while they're running during the deploy. I think I'm all right with that given the upside of keeping the database structure clean. We should quiet the workers before deploying though so that we don't get a bazillion errors in Rollbar.

Applies to the other table renames too.

Reply:
yeah we'll have to make sure to quiet them before the actual deploy!